// @vonage/audit/dist/enums.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.AuditEventTypes = void 0;
var AuditEventTypes;
(function (AuditEventTypes) {
    AuditEventTypes["USER_STATUS"] = "USER_STATUS";
    AuditEventTypes["USER_UPDATE"] = "USER_UPDATE";
    AuditEventTypes["USER_BILLING_UPDATE"] = "USER_BILLING_UPDATE";
    AuditEventTypes["USER_CREATE"] = "USER_CREATE";
    AuditEventTypes["USER_LOGIN"] = "USER_LOGIN";
    AuditEventTypes["USER_LOGOUT"] = "USER_LOGOUT";
    AuditEventTypes["USER_PRODUCT_SEARCH"] = "USER_PRODUCT_SEARCH";
    AuditEventTypes["USER_API_KEYS_UPDATE"] = "USER_API_KEYS_UPDATE";
    AuditEventTypes["ACCOUNT_SECRET_DELETE"] = "ACCOUNT_SECRET_DELETE";
    AuditEventTypes["ACCOUNT_SECRET_CREATE"] = "ACCOUNT_SECRET_CREATE";
    AuditEventTypes["ACCOUNT_UPDATE_SPAMMER"] = "ACCOUNT_UPDATE_SPAMMER";
    AuditEventTypes["ACCOUNT_UPDATE_SETTINGS_API"] = "ACCOUNT_UPDATE_SETTINGS_API";
    AuditEventTypes["NUMBER_ASSIGN"] = "NUMBER_ASSIGN";
    AuditEventTypes["NUMBER_UPDATED"] = "NUMBER_UPDATED";
    AuditEventTypes["NUMBER_RELEASE"] = "NUMBER_RELEASE";
    AuditEventTypes["NUMBER_LINKED"] = "NUMBER_LINKED";
    AuditEventTypes["NUMBER_UNLINKED"] = "NUMBER_UNLINKED";
    AuditEventTypes["APP_CREATE"] = "APP_CREATE";
    AuditEventTypes["APP_UPDATE"] = "APP_UPDATE";
    AuditEventTypes["APP_DELETE"] = "APP_DELETE";
    AuditEventTypes["APP_DISABLE"] = "APP_DISABLE";
    AuditEventTypes["APP_ENABLE"] = "APP_ENABLE";
    AuditEventTypes["IP_WHITELIST_CREATE"] = "IP_WHITELIST_CREATE";
    AuditEventTypes["IP_WHITELIST_DELETE"] = "IP_WHITELIST_DELETE";
    AuditEventTypes["AUTORELOAD_ENABLE"] = "AUTORELOAD_ENABLE";
    AuditEventTypes["AUTORELOAD_UPDATE"] = "AUTORELOAD_UPDATE";
    AuditEventTypes["AUTORELOAD_DISABLE"] = "AUTORELOAD_DISABLE";
})(AuditEventTypes = exports.AuditEventTypes || (exports.AuditEventTypes = {}));
